pandora-client-web/src/components/wiki/pages/rooms.tsx

```diff
@@ -0,0 +1,213 @@
+			</p>
+			<ul>
+				<li>You can zoom the room canvas with the mouse wheel or a pinch-to-zoom gesture.</li>
+				<li>You can drag the canvas freely to see a different part of it while zoomed in.</li>
+				<li>If you experience performance issues, you can lower/disable graphics related features in Pandora's settings.</li>
+			</ul>
+
+			<h4 id='RO_Room_ownership'>Room ownership</h4>
+			<p>
+				Rooms in Pandora are owned by one or more persons.
+			</p>
+			<ul>
+				<li>Every owner is automatically admin in the room.</li>
+				<li>Player accounts that are not owner can still be admin of a room.</li>
+				<li>Creating a new room makes you automatically owner of it.</li>
+				<li>There is a limit to how many rooms you can own. You can see your ownership limit in the room search behind the heading "My rooms".</li>
+				<li>You currently cannot add other owners to your room. This is planned.</li>
+			</ul>
+
+			<h4 id='RO_Room_deletion'>Room deletion</h4>
+			<p>
+				To permanently delete a room, you have to give up ownership over it. The room is automatically deleted if it has no other owners.
+				Removal of ownership can be done in the chat room list when clicking on the room or in the room administration screen when inside the room.
+			</p>
+
+			<h4 id='RO_Room_persistence'>Room persistence</h4>
+			<p>
+				Rooms in Pandora are not lost when they are empty. Every room will stay as it was when it was left, including characters that are offline inside it.
+			</p>
+			<ul>
+				<li>All room settings such as name, description, admin and ban lists will be persistent.</li>
+				<li>All items in the room will stay as and where they are.</li>
+				<li>If characters will disconnect or log off inside a room, they will stay in it, until they are removed from the room manually or automatically.</li>
+			</ul>
+
+			<h4 id='RO_Room_visibility'>Room visibility</h4>
+			<p>
+				Rooms in Pandora can be public or private, which affects who can see them.
+			</p>
+			<ul>
+				<li>The default when creating a new room is private.</li>
+				<li>Public room are only visible for other players when there is an admin online inside the room.</li>
+				<li>Private rooms are currently only visible in the room search for admins and owners of that room and cannot be found otherwise.</li>
+				<li>Accounts can still see rooms they are banned from</li>
+				<li>Certain information about a room normally visible from the outside, such as who is inside, is not shown when the room is password protected or the viewing account is on the room's ban list.</li>
+			</ul>
+
+			<h4 id='RO_Room_access'>Room access</h4>
+			<p>
+				Rooms in Pandora that are visible can be joined unless they are password protected.
+			</p>
+			<ul>
+				<li>Both public and private rooms can be password protected.</li>
+				<li>Admins do not need to know and enter the password to join a password protected room.</li>
+				<li>When an account is banned from a room, it cannot join it.</li>
```

Comment:
Why "they"? "account" is singular and an object.

Reply:
since to me the account feels like a person, thereby a "they" feels more appropriate

Reply:
Even then, it would still be a singular person not a plural, but how about I write simply:
`When a user's account is banned from a room, the user cannot join it.`

Reply:
sure~~

(they is singular if it refers to a person in a gender natural way)